overlord/restart/restart.go

```diff
@@ -252,6 +258,34 @@ func setWaitForSystemRestart(chg *state.Change) {
 	chg.Set("wait-for-system-restart", true)
 }
 
+// notifyRebootRequiredClassic will write the
+// /run/reboot-required{,.pkgs} marker file
+func notifyRebootRequiredClassic(task *state.Task) error {
+	var rebootRequiredSnap string
+	if err := task.Get("reboot-required-snap", &rebootRequiredSnap); err != nil {
+		return fmt.Errorf("cannot get snap that triggered the reboot: %v", err)
+	}
+
+	// XXX: This will be replaced once there is a better way to
+	// notify about required reboots.  See
+	// https://github.com/uapi-group/specifications/issues/41
+	//
+	// For now call the update-notifier script with similar inputs
+	// as apt/dpkg.
+	nrrPath := filepath.Join(dirs.GlobalRootDir, "/usr/share/update-notifier/notify-reboot-required")
+	if osutil.FileExists(nrrPath) {
+		cmd := exec.Command(nrrPath)
+		cmd.Env = os.Environ()
+		cmd.Env = append(cmd.Env, fmt.Sprintf("DPKG_MAINTSCRIPT_PACKAGE=snap:%s", rebootRequiredSnap))
+		cmd.Env = append(cmd.Env, "DPKG_MAINTSCRIPT_NAME=postinst")
```

Comment:
nitpick: this can be done with one call
```suggestion
		cmd.Env = append(os.Environ(),
		  fmt.Sprintf("DPKG_MAINTSCRIPT_PACKAGE=snap:%s", rebootRequiredSnap),
		  "DPKG_MAINTSCRIPT_NAME=postinst")
```

Reply:
Thanks, updated